public/js/form.js

```diff
@@ -0,0 +1,61 @@
+import { renderInput } from "./renderInput";
+
+const mainForm = document.querySelector("#js-form");
+const addBtn = document.querySelector("#js-button-add");
+const formField = mainForm.querySelector("#js-fields-wrapper");
+
+const MAX_INPUT_COUNT = 4;
+
+const getAllInputs = () => {
+  return mainForm.querySelectorAll('[data-role="input"]');
+};
+
+const getInputsValue = () => {
+  return mainForm.querySelector('[data-role="input"]').value;
```

Comment:
teraz pobierasz wartość tylko z pierwszego `inputa`
jeśli chcesz pobrać wartości ze wszystkich inputów, to pewnie musiałbyś zrobić coś takiego:

```js
const getInputsValue = () => {
const inputs = mainForm.querySelectorAll('[data-role="input"]
const inputsArray = Array.from(inputs);

return inputsArray.map(input => input.value); // to CI powinno zwrócic tablicę wartości: ["wartosc z 1 inputa", "wartosc z 2 inputa", ...]
}
```